.travis.yml

```diff
@@ -40,8 +40,8 @@ install:
   - travis_install_nightly 8.0 # only used if VERSION not set in env
 
 script:
-  - self_tests
+  - coverage run --rcfile=.coveragerc_mqt ./travis/self_tests
   - travis_run_tests 8.0  # only used if VERSION not set in env
```

Comment:
I kind of need to cover that line, self_tests doesn't cover a lot.

Keep that in mind for your review